src/main/java/seedu/address/commons/util/CollectionUtil.java

```diff
@@ -32,4 +32,14 @@ public static void requireAllNonNull(Collection<?> items) {
     public static boolean isAnyNonNull(Object... items) {
         return items != null && Arrays.stream(items).anyMatch(Objects::nonNull);
     }
+
+    public static boolean isAllNonNull(Object... items) {
```

Comment:
What is this for exactly?

Reply:
> 

It is just to add the use of assertions because they look for that...